### DISCUSSION TITLE
Scale gaussian process lengthscale based on dimensions?

### DISCUSSION THREAD

**Discussion by cTatu** [SimonBlanke/Gradient-Free-Optimizers#52] [Ideas]
Hi, I discovered this library and saw that sequential Model-Based Optimization algorithms are limited to ~30 dimensions.
Maybe this work [(Vanilla Bayesian Optimization Performs Great in High Dimensions)](https://www.youtube.com/watch?v=Hfsqj9v1gv4) could help remove this restriction for bayesian optimization.
[Official implementation here](https://github.com/hvarfner/vanilla_bo_in_highdim)

**Comment by SimonBlanke** [comment]
Sounds interesting, but haven't gotten around watching the video yet. I'll get back to you soon.
